crates/wash-lib/src/build/component.rs

```diff
@@ -434,6 +434,9 @@ async fn generate_tinygo_bindgen(project_dir: impl AsRef<Path>) -> Result<()> {
     let mut command = tokio::process::Command::new("go");
     let result = command
         .args(vec!["generate", "./..."])
+        // NOTE: this can be removed once upstream merges verbose flag
+        // https://github.com/bytecodealliance/wasm-tools-go/pull/214
```

Comment:
Would it be better to capture the output in a buffer and print it only if the `go generate` fails?

Reply:
Definitely, will add this!